.openshift/datasync-app-template.yml

```diff
@@ -78,8 +134,13 @@ objects:
           - containerPort: 4000
             protocol: TCP
           resources: {}
+          volumeMounts:
+            - name: config
+              mountPath: "/usr/src/model"
```

Comment:
This path is based on https://github.com/aerogear/datasync-starter/pull/444/commits/550f98ace06e5b0c059dc71cce9e41ed213b4564#diff-9b7885c8f82945595429185389c21936R19